File: Material_de_apoyo/Sistemas_Distribuidos-master 29 nov/Practica 7 29_Agosto_2018/Ejercicio_4/Ejercicio_4.java
import java.lang.Math;

public class PruebaRendimiento
{
    public static void main(String[] args)
     {
       
     	double i = 0;
		int max = 100000000;

		double seno;
		double coseno;
		double tangente;
		double logaritmo;
		double raizCuadrada;


		while(i < max){

			coseno = coseno + Math.cos(i);
			seno = seno + Math.sin(i);
			tangente = Math.tan(i);
			logaritmo = logaritmo + Math.log(i);
			raizCuadrada = raizCuadrada + Math.sqrt(i);

		i++;

	}

	System.out.printl("Valor del seno : " + seno );
	System.out.printl("Valor del coseno : " + coseno );
	System.out.printl("Valor del tangente : " + tangente );
	System.out.printl("Valor del logaritmo : " + logaritmo );
	System.out.printl("Valor del raizCuadrada : " +raizCuadrada );




    }
}

File: Material_de_apoyo/Sistemas_Distribuidos-master 29 nov/Practica 7 29_Agosto_2018/Ejercicio_4/Ejercicio_4.c
#include <stdlib.h>
#include <stdio.h>
#include <math.h>

int main(){


	double i = 0;
	int max = 100000000;

	double seno;
	double coseno;
	double tangente;
	double logaritmo;
	double raizCuadrada;


	while(i < max){

		coseno = coseno + cos(i);
		seno = seno + sin(i);
		tangente = tan(i);
		logaritmo = logaritmo + log(i);
		raizCuadrada = raizCuadrada + sqrt(i);

		i++;

	}


	printf("Valor del seno %f\n: " , seno );
	printf("Valor del coseno %f\n: " , coseno );
	printf("Valor del tangente %f\n: " , tangente );
	printf("Valor del logaritmo %f\n: " , logaritmo );
	printf("Valor del raizCuadrada %f\n: " ,raizCuadrada );





	return 1;
}